arangod/Replication2/StateMachines/Document/DocumentLeaderState.cpp

```diff
@@ -201,9 +208,15 @@ auto DocumentLeaderState::recoverEntries(std::unique_ptr<EntryIterator> ptr)
     }
 
     auto abortAllResult = data.transactionHandler->applyEntry(abortAll);
-    TRI_ASSERT(abortAllResult.ok());
+    TRI_ASSERT(abortAllResult.ok()) << abortAllRes.result();
```

Comment:
This checks the result of the local `transactionHandler->applyEntry`, but prints the result of `replicateOperation`. Is that intentional? The variable names are a little ambiguous here.

Reply:
Thanks for noticing that! Fixed in https://github.com/arangodb/arangodb/pull/19585/commits/fb8206f18f0b2c8139b0bc78212c3766d51c810d